src/client/pac/PacWrapper.ts

```diff
@@ -70,6 +70,11 @@ export class PacInterop implements IPacInterop {
 
             const env : NodeJS.ProcessEnv = {...process.env, 'PP_TOOLS_AUTOMATION_AGENT': this.context.automationAgent };
 
+            // If the VS Code telemetry is disabled, disable telemetry on the PAC backing the Extension's UI
+            if (!vscode.env.isTelemetryEnabled) {
+                env['PP_TOOLS_TELEMETRY_OPTOUT'] = 'true';
```

Comment:
Love this. it makes it so we don't also turn off telemetry for other installations of Pac CLI on the same machine.